extensions/transactions/create-replace-update-delete/standard/clause_6_create-replace-delete.adoc

```diff
@@ -67,6 +67,15 @@ The API Description Document describes the HTTP status codes generated by that A
 
 See <<OAFeat-1,OGC API - Features - Part 1: Core>>, section link:http://www.opengis.net/doc/IS/ogcapi-features-1/1.0#cross_origin[Support for cross-origin requests], about the importance of supporting cross-origin requests, typically via link:https://en.wikipedia.org/wiki/Cross-origin_resource_sharing[Cross-origin resource sharing (CORS)].
 
+==== Schemas
+
+This specification makes no assumptions about any schema constraints that a server may impose upon incoming features.  The intent is to support both "schema-less" servers (such as those back-ended by document data stores) as well as servers that have underlying schema constraints (such as those back-ended by an RDBMS).
+
+In the "schema-less" case, as long as the incoming feature meets the requirements of the declared media type (e.g. a valid GeoJSON feature) then the resource should be created/updated with a 20x status response.  
+
+In the case where the server imposes schema constraints, the server should publish schemas in order to help clients create requests that are not rejected.  How those schemas are published is out of scope for this standard although other parts of the OGC API Features suite of standard may address this topic.
```

Comment:
> How those schemas are published is out of scope for this standard ...

This is not consistent with section `#feature-representations` which has requirements about schemas. But we could also merge this now as we will have to update all mentions of schemas as a result of #740 and also #612. We might also move all schema discussions directly to the [new Schemas extension](https://github.com/opengeospatial/ogcapi-features/tree/master/proposals/schemas), that this sentence alludes to.